src/test/unit_tests/test_nc_nsfs_cli.js

```diff
@@ -530,6 +531,33 @@ mocha.describe('manage_nsfs cli', function() {
             }
         });
 
+        mocha.it('cli account add - uid=0, gid!=0', async function() {
+            const action = ACTIONS.ADD;
+            const account_name = 'uid_is_0';
+            const options = { name: account_name, email: account_name, uid: 0, gid: 1001 };
```

Comment:
So it won't confuse I would add a variable for the account email, for example:
```suggestion
            const account_name = 'uid_is_0';
            const account_email = `${account_name}@noobaa.com`
            const options = { name: account_name, email: account_email, uid: 0, gid: 1001 };
```